src/main/java/io/swagger/codegen/v3/generators/DefaultCodegenConfig.java

```diff
@@ -252,9 +254,17 @@ public void processOpts() {
             this.setUseOas2(Boolean.valueOf(additionalProperties.get(CodegenConstants.USE_OAS2).toString()));
         }
 
+        if (additionalProperties.containsKey(NOT_NULL_JACKSON_ANNOTATION)) {
+            this.setNotNullJacksonAnnotation(Boolean.valueOf(additionalProperties.get(NOT_NULL_JACKSON_ANNOTATION).toString()));
+        }
+
         setTemplateEngine();
     }
 
+    protected void setNotNullJacksonAnnotation(Boolean notNullJacksonAnnotation) {
+        this.notNullJacksonAnnotation = notNullJacksonAnnotation;
+    }
```

Comment:
this is specific to a generator, not to the whole set, DefaultCodegen must be left unchanged, you would code this into the Java generator

Reply:
done